ankimani/StackOverflow-lite
# #162559700 sign up .html

#### What does this PR do?
It will Add An updated sign up.html form
#### Description of Task to be completed?
The Following Tasks Should be completed:
1.Update the sign up.html file
2. Add styling to the sign up.html file
#### How should this be manually tested?
This can be tested as follows:
1. on GitHub via https://github.com/ankimani/StackOverflow-lit Clone the Repository to your desired folder in your local repository 
2.Then Open the file sign up.html using your desired browser and check whether the contents of the UI displays Properly in the browser. Also test for Responsiveness by minimizing the browser to test display on different devices screen. Check whether there is a link to the log in in the sign up page
#### What are the relevant pivotal tracker stories?
#162559700
